pkg/plugins/policies/meshhttproute/api/v1alpha1/validation.go

```diff
@@ -108,6 +109,32 @@ func validateMethod(match *Method) validators.ValidationError {
 	return validators.ValidationError{}
 }
 
+func validateHeaders(headers []common_api.HeaderMatch) validators.ValidationError {
```

Comment:
you wrote in [docs](https://github.com/kumahq/kuma/pull/5943/files#diff-23a5b7b1e404474dba0866ac390ea41b771e9b0d04cfd7681838700900b90506R31) that header should be lovercase, can we validate it here? so that users won't make this mistake?

Reply:
oh wait it's already there

Reply:
https://github.com/kumahq/kuma/blob/967bc8ef7c5dd5c9332307515593fb06c7aeb3f0/api/common/v1alpha1/header.go#L6

Reply:
Note that I only moved those types from the meshretry API